buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -131,6 +133,7 @@ class ClusterFormationTasks {
     /** Adds a task to extract the elasticsearch distribution */
     static Task configureExtractTask(String name, Project project, Task setup, NodeInfo node) {
         List extractDependsOn = [project.configurations.elasticsearchDistro, setup]
+        // TODO assert that project.configurations.elasticsearchDistro.singleFile is from a project and not the cache
```

Comment:
Not sure what you mean by "do the test"? We dont' run the distribution tests for external plugins. But this code to configure and start an ES cluster is used by all integ tests, as well as the run command, for plugins.